router/forwarder/forwarder.go

```diff
@@ -129,9 +131,15 @@ func (forwarder *Forwarder) Route(route *ctrl_pb.Route) {
 		circuitFt = newForwardTable()
 	}
 	for _, forward := range route.Forwards {
+		// the initiating Xgress isn't established until we return from routing, so that won't exist yet.
+		// Everything else (links and the terminating xgress) should exist at the time of routing
+		if !strings.HasSuffix(forward.DstAddress, "I") && !forwarder.HasDestination(xgress.Address(forward.DstAddress)) {
```

Comment:
magic "I" constant? :/ why not make it a legit constant? just the sort of thing i notice in PRs

Reply:
Makes sense. I'll have to make a  common consts package for it

Reply:
Removed constants and used a type field in the route message instead. Felt cleaner